vue/src/AdminPage/AdminPage.vue

```diff
@@ -125,10 +125,9 @@ export default defineComponent({
       })}`;
     },
     logImporterNoteText() {
-      const link = 'https://github.com/matomo-org/matomo/tree/master/tests#testing-data';
       return translate(
         'VisitorGenerator_LogImporterNote',
-        `<a href="${link}" rel="noreferrer noopener" target="_blank">`,
+        externalLink('https://github.com/matomo-org/matomo/tree/master/tests#testing-data'),
```

Comment:
@bx80 No parameter is appended here

Reply:
This is a github domain so campaign parameters won't be added.